src/main/java/org/sonarlint/intellij/core/SonarLintEngineFactory.java

```diff
@@ -53,6 +53,7 @@ ConnectedSonarLintEngine createEngine(String connectionId, boolean isSonarCloud)
     var configBuilder = isSonarCloud ? ConnectedGlobalConfiguration.sonarCloudBuilder() : ConnectedGlobalConfiguration.sonarQubeBuilder();
     configBuilder
       .addEnabledLanguages(EmbeddedPlugins.getEnabledLanguagesInConnectedMode().toArray(new Language[0]))
+      .enableHotspots()
```

Comment:
In the case of SonarCloud we should not enable hotspots for now

Reply:
This `enableHotspots` flag means "enable if the server permits it", so anyway we check if it's SC [a bit later](https://github.com/SonarSource/sonarlint-core/blob/master/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/hotspot/HotspotApi.java#L59) down the calls. I'm tempted to leave it like this, the day we can get hotspots from SC we don't have to change SLI